app/src/main/java/ch/epfl/skysync/models/reports/Report.kt

```diff
@@ -0,0 +1,13 @@
+package ch.epfl.skysync.models.reports
+
+import ch.epfl.skysync.models.flight.Vehicle
+import java.sql.Time
+
+interface Report {
+  val begin: Time
```

Comment:
Here you are using the `Time` class which is part of the sql package, instead consider using the `java.util.Date` class which is also used in the rest of the application

Reply:
You're right it is easier to reuse it elsewhere